ivy_models/mlpmixer/mlpmixer.py

```diff
@@ -0,0 +1,171 @@
+import ivy
+from ivy_models.base import BaseSpec, BaseModel
+
+
+class MLPMixerSpec(BaseSpec):
+    def __init__(
+        self,
+        vol=72,
+        inp_dim=3,
+        num_classes=10,
+        num_blocks=2,
+        patch_size=16,
+        hidden_dim=128,
+        token_mlp_dim=64,
+        channel_mlp_dim=32,
+        data_format="NHWC",
+    ):
+        super(MLPMixerSpec, self).__init__(
+            vol=vol,
+            inp_dim=inp_dim,
+            num_classes=num_classes,
+            num_blocks=num_blocks,
+            patch_size=patch_size,
+            hidden_dim=hidden_dim,
+            token_mlp_dim=token_mlp_dim,
+            channel_mlp_dim=channel_mlp_dim,
+            data_format=data_format,
+        )
+
+
+class MLPBlock(BaseModel):
```

Comment:
sorry, might be a misunderstanding here, only the main model class needs to be inherited from `BaseModel`, the other should just be `ivy.Module`